docs/src/components/LiveCodeExamples/Button/LiveButton.tsx

```diff
@@ -2,6 +2,7 @@ import { Button } from '@pluralsight/react'
 import { button } from '@/styled-system/recipes'
 import { css, cx } from '@/styled-system/css'
 import LiveCodeEditor from '../LiveCodeEditor'
+import { PlaceholderIcon } from '@pluralsight/icons'
```

Comment:
```suggestion
import { PlaceholderIcon } from '@pluralsight/react/icons'
```
This should work but is being funky. Just want to test.

Reply:
sure thing!